DPGAnalysis/HcalTools/plugins/CMTRawAnalyzer.cc

```diff
@@ -9626,7 +9712,7 @@ void CMTRawAnalyzer::fillDigiAmplitudeQIE11(QIE11DataFrame qie11df) {
   int c4 = 0;
   double errorBtype = 0.;
 
-  int TSsize = 10;  // sub= 1 HB
+  int TSsize = 8;  // sub= 1 HB
   if (sub == 2)
     TSsize = 8;  // sub = 2 HE
```

Comment:
This doesn't change what above, also taking into account that TSsize is redefined once again at line 935 right below (but see previous comments for a more rational usage of `TSsize`)